bubblewrap.c

```diff
@@ -1443,6 +1451,26 @@ setup_newroot (bool unshare_pid,
 
           break;
 
+        case SETUP_MOUNT_SYS:
+          if (ensure_dir (dest, 0755) != 0)
+            die_with_error ("Can't mkdir %s", op->dest);
+
+          if (unshare_pid || opt_pidns_fd != -1)
```

Comment:
sysfs is also about unshare_net. Also check this with your mount is too revealing error. There is a difference between `--unshare-user --share-net` and `--unshare-user --unshare-net` for sysfs.